thirdeye-ui/src/app/components/alert-wizard-v3/anomalies-filter-panel/anomalies-filter-panel.utils.ts

```diff
@@ -53,6 +53,18 @@ export const getAvailableFilterOptions = (
                     filteringPropertiesBySubStep;
             }
         }
+        if (property.step === "DETECTION") {
+            if (
+                property.name !== "lookback" &&
```

Comment:
is the intent to not show properties that are already shown in the main flow ? 
as implemented this won't work for other templates